debian/rules

```diff
@@ -8,6 +8,10 @@ export PYBUILD_NAME=eos-data-distribution
 
 override_dh_install:
 	dh_install --fail-missing
+	mkdir -p debian/edd-avahi-monitor/usr/bin
```

Comment:
Oh, it's because you're setting `PYBUILD_NAME` above. That breaks the multiple package setup in pybuild since it makes the destdir debian/python-eos-data-distribution instead of debian/tmp. Per pybuild(1), you just need to have debian/python-eos-data-distribution.install, which you have.